pkg/encrypt/server-side.go

```diff
@@ -90,6 +95,14 @@ type ServerSide interface {
 // Using SSE-S3 the server will encrypt the object with server-managed keys.
 func NewSSE() ServerSide { return s3{} }
 
+// NewSSEKMS returns a new server-side-encryption using SSE-KMS and the provided Key Id.
+func NewSSEKMS(keyID string) ServerSide { return kms{key: keyID, context: nil} }
```

Comment:
Maybe we don't need two functions right now - removing a function breaks API while we can always add if requested.

Reply:
Ok, perfect, I'll leave one only.